packages/core/src/components/TabbedLayout/TabbedLayout.tsx

```diff
@@ -28,6 +28,7 @@ type SubRoute = {
   path: string;
   title: string;
   children: JSX.Element;
+  tabComponent?: React.ElementType;
```

Comment:
We could alternatively just pass through a (Partial) tabProps object? That would allow more flexibility if desired (not sure about that tbh)

Reply:
Yup, that's probably a better approach in case there's need for a bit more flexibility. I added the object in with TabProps type that is extended with the hidden component prop in MUI.